os_win/include/extra/gc.h

```diff
@@ -0,0 +1,2 @@
+/* This file is installed for backward compatibility. */
+#include <gc/gc.h>
```

Comment:
Copy-pasting the source of Boehm GC doesn't look like a good idea at all maintenance-wise. We need to figure out a way to automatically download the latest source from the official repo in the build instead.